test/command_callback/test_c_checkpatch_command_callback.vader

```diff
@@ -0,0 +1,51 @@
+Before:
+  Save g:ale_c_checkpatch_executable
```

Comment:
Rebase your code and look at how linter tests are now written. See `:help ale-development-linter-tests` in the latest version of ALE for information on how to write linter tests more easily.